include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -141,6 +141,18 @@ class LAPACKFullMatrix : public TransposeTable<number>
   void add (const number                    a,
             const LAPACKFullMatrix<number> &A);
 
+  /**
+   * Perform a rank-1 update of a symmetric matrix
+   * $ A \leftarrow A + a \, \rm v \rm v^T $.
+   *
+   * This function also works for Cholesky factorization. Updating ($a>0$) is
+   * performed via Givens rotations, whereas downdating ($a<0$) via hyperbolic
+   * rotations. Note that the later case might lead to a negative definite
+   * matrix in which case the error will be thrown.
```

Comment:
-> an error will be thrown (because Cholesky factorizations are only valid for symmetric and <i>positive definite</i> matrices)